frontend/packages/console-dynamic-plugin-sdk/src/utils/k8s/hooks/useK8sWatchResource.ts

```
@@ -64,8 +64,10 @@ export const useK8sWatchResource: UseK8sWatchResource = (initResource) => {
     }
     if (!resourceK8s) {
       const data = resource?.isList ? [] : {};
-      return modelsLoaded && !k8sModel
-        ? [data, true, new NoModelError()]
+      return modelsLoaded
+        ? !k8sModel
+          ? [data, true, new NoModelError()]
+          : [data, true, undefined]
```

Any reason you decided the suppress the error?